### PR TITLE
[geographiclib] Update to version 1.52

### PR DESCRIPTION
**Pull request by cffk**
**Describe the pull request**

- #### What does your PR fix?  
  Update to version 1.52

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  yes

### CODE REVIEW

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/geographiclib/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "geographiclib",
-  "version-string": "1.51",
+  "version-string": "1.52",
```
[the remark this answers is not in the document]
Comment (cffk) [reply ports/geographiclib/vcpkg.json]: Thanks, done.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
@cffk ,Have you tested the tool feature?

**Comment by cffk** [comment]
> @cffk ,Have you tested the tool feature?

Yes, this works.

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**